velox/serializers/PrestoSerializer.cpp

```diff
@@ -4261,6 +4190,113 @@ void PrestoVectorSerde::deserializeSingleColumn(
   *result = row->childAt(0);
 }
 
+void PrestoVectorSerde::serializeColumn(
```

Comment:
@pramodsatya : Is this new code ?

Reply:
No @aditi-pandit, this function was present [here](https://github.com/facebookincubator/velox/blob/763c19c0c767035ce03573ca2dc852b1303eeb6f/velox/serializers/PrestoSerializer.cpp#L2256) and is now added to `PrestoVectorSerde`. 
The second `serializeColumn` function was present [here](https://github.com/facebookincubator/velox/blob/763c19c0c767035ce03573ca2dc852b1303eeb6f/velox/serializers/PrestoSerializer.cpp#L2256), and is also added to `PrestoVectorSerde`.